prime-router/src/main/kotlin/Element.kt

```diff
@@ -307,24 +301,30 @@ data class Element(
         normalizedValue: String,
         format: String? = null,
     ): String {
-        val cleanedNormalizedValue = normalizedValue.trim()
-        if (cleanedNormalizedValue.isEmpty()) return ""
+        // trim the normalized value down to null and if it is null return empty string
+        val cleanedNormalizedValue = normalizedValue.trimToNull() ?: return ""
         val formattedValue = when (type) {
             // sometimes you just need to send through an empty column
             Type.BLANK -> ""
-            Type.DATE -> {
-                if (format != null) {
-                    val ta = parseDate(cleanedNormalizedValue)
-                    getDate(ta, format)
-                } else {
+            Type.DATETIME -> {
+                try {
```

Comment:
Can we wrap the try/catch in some parseOrNull pattern?

Reply:
I will spec out a tryParseOrRaw method